### PR TITLE
Create honorarium policy

### PR DESCRIPTION
**Pull request by emmyft**

### CODE REVIEW

**Comment by emmyft** [comment]
Would appreciate 👀 over! (I can't assign reviewers here... cc @yochannah @pazbc ) Thank you!

File: payments/honorarium.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- Participation in other OLS events as a trainer, speaker, mentor, etc
+- Participation in other OLS activities, e.g. hiring
+- Participation/contribution in an event on behalf of OLS
+
+No honoraria from OLS can be offered to currently engaged OLS employees, independent contractors, or consultants. The recipient of the honorarium is not permitted to set the honorarium amount. The arrangement between the recipient and OLS should not involve an existing contract. 
+
+Currently, OLS may offer:
+- 250 USD for expert speakers in an OLS cohort;
+- 500 USD for mentoring in an OLS cohort;
+- 50 USD/hour, for facilitation in an OLS cohort;
```
Comment (pherterich) [inline payments/honorarium.md:18]: Not sure the comma is needed here

File: payments/honorarium.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Generally, OLS community members who have been asked to perform a paid task by OLS community organisers are paid at 50 USD/hour.
+
+Individuals who are offered an honorarium can choose to donate the honorarium back to OLS’s work or to a charity of their choice. They can also choose to ask OLS to pay for a training session/conference fee up to the value of the payment - this offer expires in a year after the submission of the honorarium payment form, if unused.
```
Comment (pherterich) [inline payments/honorarium.md:22]: this offer expires in a year after the submission -> this offer expires one year (=12 months) after the submission

Rewording might make it more clear?

**Review by pherterich — APPROVED** [review]
Minor comments, otherwise this is good to go!

**Review by pherterich — APPROVED** [review]

**Review by yochannah — APPROVED** [review]

**Pull request merged**